### DISCUSSION TITLE
유니티 프로젝트 폴더 구조 모범 사례

### DISCUSSION THREAD

**Discussion by fkdl0048** [The-Day-In-1984/1984_Project#14] [General]
https://sam-16930.medium.com/unity-project-structure-a694792cefed

@kyoonge 장현님이 프로젝트 폴더 말씀해주셔서 모범사례? 조사해봤는데 더 좋은 예시나 가져가고 싶은 내용 있으시면 쓰레드로 달아주세요

**Comment by kyoonge** [comment]
![image](https://user-images.githubusercontent.com/96893605/225242111-27662d9d-8ba6-43a2-a2e4-f7a792dbb600.png)

오 완전 깔끔하고 좋네여 이걸로 가시죠
